backend/services/workflows/tests/docker-compose.yml

```diff
@@ -5,9 +5,6 @@ services:
     extends:
       file: ../../../tests/docker-compose.yml
       service: acceptance-tester
-    command: # FIXME(QA-733): Align dockerfile with open source
-      - -k
-      - not test_create_artifacts
```

Comment:
I kind of think that this dance with the open source tests in the ent and vice-versa is pointless. os code, os tests, os repo. ent code, ent tests, ent repo. perhaps it would be easier. idle comment here totally.